radis/io/hitran.py

```diff
@@ -1285,6 +1340,7 @@ def register(self):
 # TODO: implement parallel=True for all isotopes ?
 def fetch_hitran(
     molecule,
+    extra_params=None,
```

Comment:
Add a description in docstrings.

Also can you add a quick example showing how to use it?